force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -176,7 +214,10 @@ public with sharing class VariablesDataTypesOperators {
      * @return The average of the three numbers.
      */
     public static Double averageOfThreeNumbers(Integer a, Integer b, Integer c) {
-        return null; // Replace null with the variable you used to store the result
+       double firstNumber = a;
```

Comment:
Datatypes should be capitalized. Also, just use the input parameters and don't reassign to new variables